packages/wdio-allure-reporter/tests/allureFeatures.test.js

```diff
@@ -462,6 +461,21 @@ describe('hooks handling', () => {
         expect(endStep).toHaveBeenCalledTimes(1)
         expect(endStep.mock.results[0].value).toBe('failed')
     })
+
+    it('should ignore mocha all hooks if hook passes', () => {
+        reporter.allure = allureInstance()
+        reporter.onHookEnd({ title: '"before all" hook', parent: 'foo' })
+
+        expect(endStep).toHaveBeenCalledTimes(0)
```

Comment:
1) not `endStep` but `endCase`
2) also please add test that `reporter.onHookStart({ title: '"before all" hook', parent: 'foo' })` startCase isnt invoked